### DISCUSSION TITLE
Q&A on connecting a wallet to LND

### DISCUSSION THREAD

**Discussion by urPlaceOrMine** [lightningnetwork/lnd#6640] [Ideas]
Hello - 

First off and foremost I am new to your world.

Working through a[ tutorial](https://medium.com/@stopanddecrypt/a-complete-beginners-guide-to-installing-a-lightning-
node-on-linux-2021-edition-ece227cfc35d) on how to set up LND.  I have LND connected to my Bitcoind install.  I am 
struggling with connecting LND to a wallet.  They author suggest ZAP-IOS which seems to be a little defunct IMHO.

Which brings me to this fundamental question.  What wallets can be connected to LND?  Any wallet that support the satoshi(?).  I want to be able to send and receive funds to the wallet before a get too far down a bit-bucket road of no return.  

I see some instructions on using the command line to connect wallets is that simply the easiest

Thank you for taking time to help join join your community. 

ur

**Comment by guggero** [comment]
Hi

I would highly recommend [Zeus](https://docs.zeusln.app/) as that is being improved constantly. I personally switched over from Zap to Zeus after the development on Zap was discontinued.

What you need to do is enable REST on your node, then use `lndconnect` as described in the tutorial you linked. You might need to use the `--port=xxxx` flag in `lndconnect` to point to your REST port (it uses the gRPC port by default).

There's also a Telegram channel where you can request assistance: https://t.me/zeusLN

**Reply by urPlaceOrMine** [reply]
Great.  Thank you for your input.  I will Zeus try it out.
